### PR TITLE
Dashboard skeleton#4

### PR DESCRIPTION
**Pull request by MillanAir**
- Added assets folder containing images and logo for the site
- Added Navbar with tabs and placeholders
- Added placeholder pages for Shopping List, Friends and Notifications

Package.json files looks very huge, is it normal? and is it a good idea to commit this file and package-lock.json next time?
image

### CODE REVIEW

File: client/src/components/header/MenuTabs.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import {Link} from 'react-router-dom';
+
+function TabPanel(props) {
```
Comment (jessdesr) [inline client/src/components/header/MenuTabs.js]: What is TabPanel used for?

**Review by jessdesr — COMMENTED** [review]
In order to resolve the conflicts, you'll need to merge dev into your branch locally

**Review by jessdesr — COMMENTED** [review]
Good work, I think there's some additional code here (maybe from a googled solution?) that you can get rid off since it's not being used.

File: client/src/components/header/MenuTabs.js (hunk 1 above)
Comment (MillanAir) [reply client/src/components/header/MenuTabs.js]: There was extra unused code in the file which is removed.

**Pull request merged**